Sources/Stripe/Model.swift

```diff
@@ -130,7 +130,7 @@ public struct Coupon: Equatable {
     public var formattedDescription: String {
       switch self {
       case let .amountOff(amountOff):
-        return "$\(amountOff) off"
+        return "$\(Int(amountOff.map(Double.init).dollars.rawValue)) off"
```

Comment:
we haven't used a $ amount coupon yet, but if we had it would have shown something like "$500 off" instead of "$5 off".